portal-api/src/v1/section-status/about.js

```diff
@@ -1,15 +1,22 @@
 const submissionDetailsRules = require('../validation/submission-details-rules');
+const { FACILITIES } = require('../../constants');
 
-const aboutSupplyContractStatus = (submissionDetails) => {
-  if (submissionDetails.status === 'Not started') {
-    return 'Not started';
-  }
+/**
+ * Determines the supply contract status based on the submission details.
+ * @param {Object} submissionDetails - An object containing details of a submission.
+ * @returns {string} - The supply contract status.
+ */
+const aboutSupplyContractStatus = async (submissionDetails) => {
+  const { status } = submissionDetails;
 
-  const validationErrors = submissionDetailsRules(submissionDetails);
-  if (Object.keys(validationErrors).length > 0) {
-    return 'Incomplete';
+  if (status === FACILITIES.DEAL_STATUS.NOT_STARTED) {
+    return FACILITIES.DEAL_STATUS.NOT_STARTED;
   }
-  return 'Completed';
+
+  const validationErrors = await submissionDetailsRules(submissionDetails);
+  const isIncomplete = Object.keys(validationErrors).length > 0;
+
+  return isIncomplete ? FACILITIES.DEAL_STATUS.INCOMPLETE : FACILITIES.DEAL_STATUS.COMPLETED;
```

Comment:
Side note going back to constants earlier in the review -- We're handling statuses here in (yet) another slightly different way.